addons/binding/org.openhab.binding.loxone/src/main/java/org/openhab/binding/loxone/core/LxWsStateUpdateEvent.java

```diff
@@ -0,0 +1,136 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.loxone.core;
+
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+
+/**
+ * An event received from Loxone Miniserver with control's state update
+ *
+ * @author Pawel Pieczul - initial commit
+ *
+ */
+class LxWsStateUpdateEvent {
+    private LxUuid uuid;
+    @SuppressWarnings("unused")
+    private LxUuid iconUuid = null;
```

Comment:
For the icon field, as it parses through the message, it has to go sequentially over fields, out of which icon is not used today by the binding. But the code for parsing is complete this way and it seems to me clean if it retrieves this field too, so it is ready for future potential use.